src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(String color,double side) {
```

Comment:
When creating a new instance of 'Square', you should avoid passing expressions directly into the constructor. Instead, create a variable to hold the value and pass the variable to the constructor for better readability.